src/components/common/helpers/workflowURL.ts

```diff
@@ -1,9 +1,9 @@
 
-export function getCDPipelineURL(appId: string, workflowId: string, ciPipelineId: string, cdPipelineId: string = null) {
+export function getCDPipelineURL(appId: string, workflowId: string, ciPipelineId: string, cdPipelineId: string = null, isWebhookParent: boolean) {
```

Comment:
Can we keep the optional arg at the end?

Reply:
Done